src/Reporter.php

```diff
@@ -73,6 +74,9 @@ public function executeBeforeFirstTest(): void
         );
 
         $this->validateProjectCode();
+        if ($this->config->getEnvironmentId() !== null) {
```

Comment:
to make the method clearer move this condition to **validateEnvironmentId**

Reply:
done

Reply:
all good, but use "early return"

Reply:
done